ui/src/app/cron-workflows/components/cron-workflow-details/cron-workflow-details.tsx

```diff
@@ -62,6 +73,16 @@ export const CronWorkflowDetails = ({match, location, history}: RouteComponentPr
 
     useEffect(() => setEdited(true), [cronWorkflow]);
 
+    useEffect(() => {
+        services.workflows.list(namespace, selectedPhases, selectedLabels, pagination).then(res => {
```

Comment:
Ideally we can filter in the request to not retrieve unnecessary workflows, although that might not be possible

Reply:
this is possible -- you can pass in a label with the Cron Workflow name: `workflows.argoproj.io/cron-workflow=<name-of-cron-workflow>`

Reply:
also should use async/await per #11740

Reply:
- Thanks for the feedback, would it be ok to change to async/await and get the data via label names like on the workflows List page?

Reply:
You can replace `selectedLabels` (the 3rd argument to [`list`](https://github.com/argoproj/argo-workflows/blob/477b3caf415d1f65f71dd366d9ebc5c04c64c099/ui/src/app/shared/services/workflows-service.ts#L42)) with the `cron-workflow` label from above